### PR TITLE
Use Ansible 2.7 which can create security groups for any project

### PR DESCRIPTION
**Pull request by priteau**
We use a stackhpc branch with a fix for https://github.com/ansible/ansible/issues/41240.

### CODE REVIEW

File: {{cookiecutter.cloud_shortname}}-config/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Limit Ansible to <2.5, since the os_quota module is broken when a Cinder
-# endpoint is not available. See
-# https://github.com/ansible/ansible/issues/41240.
-ansible>=2.6.0
+# The os_quota module is broken when a Cinder endpoint is not available. See
+# https://github.com/ansible/ansible/issues/41240. This branch of Ansible 2.7
+# contains a fix.
+-e git+https://github.com/stackhpc/ansible.git@issue-41240-2.7#egg=ansible
```
Comment (markgoddard) [inline {{cookiecutter.cloud_shortname}}-config/requirements.txt]: I proposed this PR, which doesn't require this patch: https://github.com/stackhpc/ansible-role-os-projects/pull/14. It's using the CLI directly, I'm not sure about the level of support for all quotas it provides though.

Comment (markgoddard) [reply {{cookiecutter.cloud_shortname}}-config/requirements.txt]: Actually, it just does this, so should support all quotas:

```
quota set {{ item.name }}
    {% for name, value in item.quotas.items() %} --{{ name | replace('_', '-') }}={{ value }}{% endfor %}
```

Comment (markgoddard) [reply {{cookiecutter.cloud_shortname}}-config/requirements.txt]: Now in os-projects 1.3.0

Comment (priteau) [reply {{cookiecutter.cloud_shortname}}-config/requirements.txt]: Thanks a lot @markgoddard!

**Pull request merged**